src/pages/HomePage/HomePage.jsx

```diff
@@ -0,0 +1,49 @@
+import { Navbar, Sidebar } from "../../component";
+import "./homePage.css";
+import heroImg4 from "../../Assets/heroImg4.jpg";
+import breakfast from "../../Assets/breakfast.jpeg";
+import lunch from "../../Assets/lunch.jpg";
+import snacks from "../../Assets/snacks.jpg";
+import dinner from "../../Assets/dinner.jpg";
```

Comment:
Use Absoluet imports, to make imports shorter and easier.

Reply:
i had to do absolute imports but i don't know why it wasn't working must be some problem in linking.will look into it again in next PR